modules/mobile-agents/windows/jax-rs/src/main/java/org/wso2/carbon/mdm/mobileservices/windows/services/adminoperations/beans/Encrypt.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * you may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.mdm.mobileservices.windows.services.adminoperations.beans;
+
+import org.wso2.carbon.mdm.mobileservices.windows.common.util.WindowsOperation;
+
+import java.io.Serializable;
+
+/**
+ * This class represents the information of encrypt operation.
+ */
+public class Encrypt extends WindowsOperation implements Serializable {
```

Comment:
"Encrypt" is usually used as a "verb". Let's please use an appropriate noun to name the class.